docs/ansible/roles/debops.docker/defaults-detailed.rst

```diff
@@ -0,0 +1,196 @@
+Default variable details
+========================
+
+.. include:: ../../../includes/global.rst
+
+Some of ``debops.docker`` default variables have more extensive configuration
+than simple strings or lists. Here you can find documentation and examples for
+them.
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _docker_ref_networks:
```

Comment:
You should add a link near the variables in the `defaults/main.yml` file to their respective documentation sections, it will be easier to reach on the documentation page.